samples/Calling/src/app/App.tsx

```diff
@@ -134,12 +138,10 @@ const App = (): JSX.Element => {
 
             /* @conditional-compile-remove(rooms) */
             if ('roomId' in callLocator) {
-              if (userId) {
-                setRole(callDetails.role as Role);
-                await addUserToRoom(userId.communicationUserId, callLocator.roomId, callDetails.role as Role);
-              } else {
-                throw 'Invalid userId!';
-              }
+              setRole(callDetails.role as Role);
+            } else {
+              // unset role when call is not a Rooms call
+              setRole(undefined);
```

Comment:
We can be silly and just set this as Presenter instead and just add a comment that a Presenter is the same as if you hadn't set a role.